.github/pull_request_template.md

```diff
@@ -1,15 +1,15 @@
 ### Description
 <!-- MANDATORY: Summarize the intent of the change in the title. Provide a text description about the issue the PR is addressing that ensures the reader understands the context, the rationale behind and catches a 1000-feet perspective of the implementation.  Enrich the description with screenshots, code blocks. Use formatting to ensure a good readability for all public audience! -->
 
-/cc <!-- MANDATORY: Assign one reviewer from top-level OWNERS file -->
-/assign <!-- MANDATORY: Assign ne approver from top-level OWNERS file -->
+/cc <!-- MANDATORY: Assign at least one reviewer from top-level OWNERS file -->
+/assign <!-- MANDATORY: Assign at least one approver from top-level OWNERS file -->
 
-/cherry-pick <!-- OPTIONAL: Declare release name for the next release branch to get this PR cherry-picked by the bot -->
+<!-- OPTIONAL: Declare release name for the next release branch to get this PR cherry-picked by the bot. Example: /cherry-pick release-x.y  -->
```

Comment:
I think the command is actually `/cherrypick`.  I don't believe it works with the dash, at least that was my experience